app/src/main/java/se/inera/intyg/minaintyg/util/html/HTMLListFactory.java

```diff
@@ -0,0 +1,18 @@
+package se.inera.intyg.minaintyg.util.html;
+
+import static se.inera.intyg.minaintyg.util.html.HTMLFactory.tag;
+
+import java.util.List;
+
+public class HTMLListFactory {
```

Comment:
Add private constructors and throw new IllegalStateException("Utility class"); if used.